pexpect/__init__.py

```diff
@@ -68,7 +68,12 @@
 
 from .exceptions import ExceptionPexpect, EOF, TIMEOUT
 from .utils import split_command_line, which, is_executable_file
-from .pty_spawn import spawn, spawnu, PY3
+
+if sys.platform == 'win32':
+    from .popen_spawn import (
+        PopenSpawn as spawn, PopenSpawnUnicode as spawnu, PY3)
+else:
+    from .pty_spawn import spawn, spawnu, PY3
```

Comment:
Ah, the reason I did that is because you can't even import the other module on `win32` because it lacks the correct built in modules.